src/front-door/azext_front_door/custom.py

```diff
@@ -861,11 +861,11 @@ def add_custom_rule_match_condition(cmd, resource_group_name, policy_name, rule_
                 selector = variable_parts[1]
 
             rule.match_conditions.append(MatchCondition(
-                match_variable=match_variable,
-                selector=selector,

```

Comment:
I know this is only whitespace, but could we just somehow revert it?

Reply:
Sure, wired thing is I copy the source text from main branch but still get this problem, let me double fix.